controllers/scoring.py

```diff
@@ -151,13 +175,16 @@ def score(self, initial_call):
             self.gripper.open()
             self.is_holding = GamePiece.NONE
             if self.gripper.get_full_open():
+                print("scored at", tm)
                 self.next_state("idle")
                 self.arm.go_to_setpoint(Setpoints.STOW)
+                if len(self.score_queue):
+                    self.score_queue.popleft()
```

Comment:
Why a queue instead of a stack? It doesn't look like we're adding to the end of the queue anywhere.